.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build PHP/TYPO3
+    runs-on: ubuntu-latest
```

Comment:
I'm not that familier with github workflows, but does smaller images like alpine exists to make it faster?

Reply:
Available GitHub-hosted runner types are:

Virtual environment | YAML workflow label
-- | --
Windows Server 2019 | windows-latest or windows-2019
Ubuntu 20.04 | ubuntu-20.04
Ubuntu 18.04 | ubuntu-latest or ubuntu-18.04
Ubuntu 16.04 | ubuntu-16.04
macOS Big Sur 11.0 | macos-11.0
macOS Catalina 10.15 | macos-latest or macos-10.15

--------------------

It seems that they are the only available runners. But it is possible to use self-hosted runners.
See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
Interesting: latest is not latest ;-)